### PR TITLE
JumpCloud Directory Insights C2C doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+The JumpCloud Directory Insights source requires you to provide the **API Key** and **Organization ID** to access the source data.
+
+- To generate the **API Key**, follow the instructions mentioned in the [Jumpcloud documentation](https://docs.jumpcloud.com/api/insights/directory/1.0/index.html#section/Authentication-and-Authorization/Authentication).
+- To generate the **Organization ID**, follow the instructions mentioned in the [Jumpcloud documentation](https://docs.jumpcloud.com/api/insights/directory/1.0/index.html#section/Authentication-and-Authorization/Authentication).
+
+## States
+
+JumpCloud Directory Insights source is a security platform that provides cloud-based security in a cost-effective manner that doesn't sacrifice security or functionality.
+When you create an JumpCloud Directory Insights source, it goes through the following stages:
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:30]:
```suggestion
When you create a JumpCloud Directory Insights source, it goes through the following stages:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+1. **Collecting**. The Source is actively collecting data from JumpCloud Directory Insights.
+
+If the Source has any issues during any one of these states, it is placed in an **Error** state.
+
+When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
+On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. You can click the text in the Health column, such as **Error**, to open the issue in Health Events to investigate.
+
+## Create JumpCloud Directory Insights Source
+
+When you create an JumpCloud Directory Insights source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:44]:
```suggestion
When you create a JumpCloud Directory Insights source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+To configure a Code42 Incydr Source:
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:46]:
```suggestion
To configure a JumpCloud Directory Insights source:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+| ThirdPartyGeneric | Normally due to an error communicating with the third-party service APIs. | Yes | The Source will retry indefinitely. | ThirdPartyGenericError |
+| FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs. | Yes | The Source will retry indefinitely. | FirstPartyGenericError |
+
+### Restarting your Source
+
+{@import ../../../reuse/restart-c2c-source.md}
+
+### JSON Configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:76]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+{@import ../../../reuse/restart-c2c-source.md}
+
+### JSON Configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+
+| Parameter | Type | Required | Description | Access |
+|:--|:--|:--|:--|:--|
+| `config` | JSON Object  | Yes | Contains the [configuration-parameters](#config-parameters) of the Source. | n/a |
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:80]:
```suggestion
| `config` | JSON Object  | Yes | Contains the [configuration parameters](#config-parameters) of the Source. | n/a |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+| `schemaRef` | JSON Object  | Yes | Use `{"type":"JumpCloud Directory Insights"}` for JumpCloud Directory Insights Source. | not modifiable |
+| `sourceType` | String | Yes | Use `Universal` for JumpCloud Directory Insights. | not modifiable |
+
+### Config parameters
+
+| Parameter | Type | Required | Description | Access |
+|:---|:---|:---|:---|:---|
+| `name` | String | Yes | Type the desired name of the Source and it must be unique per Collector. This value is assigned to the `metadata field _source`.  | modifiable |
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:88]:
```suggestion
| `name` | String | Yes | Type the desired name of the Source. It must be unique per Collector. This value is assigned to the `metadata field _source`.  | modifiable |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+JumpCloud's open directory platform unifies your technology stack across identity, access, and device management in a cost-effective manner that doesn't sacrifice security or functionality. Directory Insights gives you a clear path to view, analyze, and save user and resource activity data, in a standard format that you can query in real time.
+
+JumpCloud Directory Insights Source is used to collect Directory Insights Events from the JumpCloud platform using the REST API and send it to Sumo Logic.
+
+## Data Source
+
+The integration fetches events data using [REST API](https://docs.jumpcloud.com/api/insights/directory/1.0/index.html#section/Overview) provided by JumpCloud Directory Insight platform.
+
+## Set up and Configuration
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:20]:
```suggestion
## Setup and Configuration
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/jumpcloud-directory-insights-logo.png')} alt="jumpcloud-directory-insights-icon" width="120" />
+
+JumpCloud's open directory platform unifies your technology stack across identity, access, and device management in a cost-effective manner that doesn't sacrifice security or functionality. Directory Insights gives you a clear path to view, analyze, and save user and resource activity data, in a standard format that you can query in real time.
+
+JumpCloud Directory Insights Source is used to collect Directory Insights Events from the JumpCloud platform using the REST API and send it to Sumo Logic.
+
+## Data Source
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:16]:
```suggestion
## Data source
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+1. (Optional) **Fields**. Click the **+Add** button to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+1. In **API Key**, enter the API Key you generated from the JumpCloud Directory Insights platform.
+1. In **Organization ID**, enter the Organization ID you generated from the JumpCloud Directory Insights platform.
+1. In **Service**, select the type of logs to collect. This allows you to limit the response to just the data you want.
+1. When you are finished configuring the Source, click **Save**.
+
+### Error Types
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:60]:
```suggestion
### Error types
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+
+When Sumo Logic detects an issue it is tracked by Health Events. The following table shows the three possible error types, the reason for the error, if the source attempts to retry, and the name of the event log in the Health Event Index.
+
+| Type | Reason | Retries | Retry Behavior | Health Event Name |
+|:--|:--|:--|:--|:--|
+| ThirdPartyConfig  | Normally due to an invalid configuration. You'll need to review your Source configuration and make an update. | No retries are attempted until the Source is updated. | Not applicable | ThirdPartyConfigError  |
+| ThirdPartyGeneric | Normally due to an error communicating with the third-party service APIs. | Yes | The Source will retry indefinitely. | ThirdPartyGenericError |
+| FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs. | Yes | The Source will retry indefinitely. | FirstPartyGenericError |
+
+### Restarting your Source
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:70]:
```suggestion
### Restarting your source
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+|:---|:---|:---|:---|:---|
+| `name` | String | Yes | Type the desired name of the Source and it must be unique per Collector. This value is assigned to the `metadata field _source`.  | modifiable |
+| `description` | String  | No | Type the description of the Source. | modifiable |
+| `category` | String | No | Type the category of the source. This value is assigned to the metadata field `_sourceCategory`. | modifiable |
+| `fields` | JSON Object | No | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM. | modifiable |
+| `apiKey` | String | Yes | API Key generated from the JumpCloud Directory Insights platform. | modifiable |
+| `orgID` | String | Yes | Organization ID generated from the JumpCloud Directory Insights platform. | modifiable |
+| `service` | String | Yes | Type of logs from which you want to collect the data from. | modifiable |
+
+### JSON Example
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/jumpcloud-directory-insights-source.md:96]:
```suggestion
### JSON example
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
This article looks very good, @JV0812! I had only a few minor comments.

**Comment by JV0812** [comment]
Thanks John! I have incorporated your comments.

**Review by jpipkin1 — APPROVED** [review]
It looks good, @JV0812! Approved.

**Pull request merged**